addons/sale/wizard/res_config_settings.py

```diff
@@ -57,6 +57,18 @@ class ResConfigSettings(models.TransientModel):
     prepayment_percent = fields.Float(
         related='company_id.prepayment_percent',
         readonly=False)
+    is_stripe_supported_country = fields.Boolean(
```

Comment:
I would not copy/paste same thing which is already there in `website_payment`. I would rather move it in `payment` module or somewhere else where it can be used in both places. What do you think?